src/base/plugins/SUBTREELoader.js

```diff
@@ -743,6 +744,17 @@ export class SUBTREELoader extends LoaderBase {
 
 			}
 
+			if ( tile.__implicitRoot.implicitTiling.subdivisionScheme === 'OCTREE' ) {
+
+				const minZ = region[ 4 ];
+				const maxZ = region[ 5 ];
+
+				const sizeZ = ( maxZ - minZ ) / Math.pow( 2, tile.__level );
+				region[ 4 ] += tile.__z * sizeZ;
+				region[ 5 ] = region[ 4 ] + sizeZ;
+
+			}
```

Comment:
Would be good to add some comments here, I think, regarding what this is adjusting and how.